vertical-pod-autoscaler/pkg/updater/logic/updater.go

```diff
@@ -142,17 +142,17 @@ func (u *updater) RunOnce(ctx context.Context) {
 
 	for _, vpa := range vpaList {
 		if slices.Contains(u.ignoredNamespaces, vpa.Namespace) {
-			klog.V(3).Infof("skipping VPA object %s in namespace %s as namespace is ignored", vpa.Name, vpa.Namespace)
+			klog.V(3).InfoS("Skipping VPA object in ignored namespace", "vpa", klog.KObj(vpa), "namespace", vpa.Namespace)
```

Comment:
namespace is already present on the VPA object metadata

```suggestion
			klog.V(3).InfoS("Skipping VPA object in ignored namespace", "vpa", klog.KObj(vpa))
```